static/app/views/dashboards/manage/index.tsx

```diff
@@ -57,6 +60,16 @@ function ManageDashboards() {
   const navigate = useNavigate();
   const location = useLocation();
   const api = useApi();
+  const dashboardGridRef = useRef<HTMLDivElement>(null);
+
+  const [showTemplates, setShowTemplatesLocal] = useLocalStorageState(
+    SHOW_TEMPLATES_KEY,
+    shouldShowTemplates()
+  );
+  const [_, setGridWidth] = useState(0);
+  const [resizing, setResizing] = useState(false);
```

Comment:
A question about this! Is `resizing` meant to indicate that the user is currently dragging the window, or is it meant to indicate that new dashboards are loading, or something else? Trying to figure out if it's possible to avoid this extra state

Reply:
ya it's meant to indicate that the everything needed to resize the window is happening. Basically saying the window dimensions are changing and the endpoint is being called again to get the correct amount of dashboards.

Reply:
🤔 if it's only used to indicate that the endpoint is being called, can you use `isFetching`? It'll be true whenever the network call is in progress.

i.e.,

```jsx
  const {
    data: dashboards,
    isFetching,
    isLoading,
    isError,
    error,
    getResponseHeader,
    refetch: refetchDashboards,
  } = useApiQuery<DashboardListItem[]>(

etc.
```